composer.json

```diff
@@ -3,7 +3,8 @@
     "description": "Privacy and consent module for Altis",
     "type": "package",
     "require": {
-        "php": ">=7.1"
+        "php": ">=7.1",
+		"altis/consent-api": "^0.9"
```

Comment:
Very minor thing, looks like the use of tabs vs spaces is inconsistent in this file. Altis modules themselves are using tabs.

Reply:
👍 fixed